src/parsers.ts

```diff
@@ -1,3 +1,4 @@
+export * from '@core' // TODO keep this here for BC? or move to a dedicated core export?
```

Comment:
let me know what you'd like to do here? I kept the core exports here for backwards compatibility - I don't know if there would be much point in creating a dedicated core module export in the package?

Reply:
Let's leave this re-export here for now. I think it doesn't make much sense to move it into a separate entry-point.

Reply:
Yeah, although I am wondering why you even have the two separate exports of parsers and combinators?

It doesn't seem like parsers are much use without combinators in a parser-combinator library?

And the combinators almost definitely aren't any use without parsers? Even if you're writing all your own parser and using the combinators only, you'd at least need `run`, right?

That, plus the root package is just functions, which are tree-shakable to begin with, aren't they?

Of course, getting rid of them is a breaking change, but... zero dependents on npm... and I don't know who would use the separate modules to begin with or why?

Might be time to just consolidate the whole thing to one module for simplicity?

Reply:
Let's keep it the way it is for now, so we can move on - I'm removing the todo.

Reply:
All what you are saying makes sense, it's just that I wanted to make multiple breaking changes (get rid of multiple entry points, rename a couple of functions, change signatures) in one go to avoid bumping major version multiple times.